docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc

```diff
@@ -2,6 +2,12 @@
 
 = Retrieving topology-specific service configuration
 
+Before you migrate your databases to the {rhos_long} control plane, retrieve the topology-specific service configuration from your {rhos_prev_long} ({OpenStackShort}) environment. You need this configuration for the following reasons:
+
+* To check your current database for inaccuracies
+* To ensure that you have the data you need before the migration
+* To compare your {OpenStackShort} database with the adopted {rhos_acro} database
```

Comment:
@matbu Can you confirm that the introduction is accurate?

Reply:
Yes +1